charts/fission-all/templates/router/secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.authentication.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: router
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+type: Opaque
```

Comment:
Should we skip type here? This way user default secret method would be used while installation ?

Reply:
Fixed